joss_paper/paper.md

```diff
@@ -9,24 +9,32 @@ tags:
 authors:
   - name: Lawson Woods
     orcid: 0009-0003-0713-4167
-    affiliation: 1 
+    affiliation: [3, 4]
   - name: Hugo Macdermott-Opeskin
```

Comment:
@hmacdope Sorry to bug you again about this, but are you okay with this author order?

Reply:
Ah yes, apologies I am happy with this author order. My name has a capital `D` though. `MacDermott-Opeskin` and my affiliation is  Open Molecular Software Foundation, Davis CA`. 

Once you are ready you should ask all authors to read final manuscript, just FYI.

Reply:
Oops, my bad, I suck at capitalization. 

Do you think this is ready to to send out as a final manuscript?